pkgs/development/python-modules/mkdocs-material-extensions/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, markdown, }:
+
+buildPythonPackage rec {
+  pname = "mkdocs-material-extensions";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "bfd24dfdef7b41c312ede42648f9eb83476ea168ec163b613f9abd12bbfddba2";
+  };
+
+  doCheck = false;
```

Comment:
Please enable tests.

Reply:
there would be a circular dependency with mkdocs-material when I would enable tests here